pkgs/os-specific/linux/nix-selinux-policy/LICENSE

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
IMHO individual Nix expressions in Nixpkgs should not have different licenses.

Reply:
Custom license removed, and since there's no upstream project, I applied nixpkgs' MIT license.